.github/workflows/sync-strategies.yml

```diff
@@ -0,0 +1,26 @@
+name: Sync ClusterBuildStrategies
+on:
+  push:
+    branches:
+      - main
+    paths:
+    - 'clusterBuildStrategy/**/*.yaml'
+  pull_request:
+    branches:
+      - main
+    paths:
+    - 'clusterBuildStrategy/**/*.yaml'
+
+jobs:
+  merge:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Code
+      uses: actions/checkout@v3 
+    - name: Sync all strategies
+      run: |
+        rm -rf all-strategies.yaml
+        for file in `ls clusterBuildStrategy/**/*.yaml`
+        do 
+          cat $file >> all-strategies.yaml
+        done
```

Comment:
Can we uploaded it to GH releases after generating it 
here is an example of step
- name: Generate and upload release.yaml
      env:
        REGISTRY_PASSWORD: ${{ secrets.GITHUB_TOKEN }}
        REGISTRY_USERNAME: ${{ github.repository_owner }}
        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
      run: |
        gh release upload ${TAG} release.yaml